pages/recipes/supabase.mdx

```diff
@@ -82,140 +86,126 @@ Started supabase local development setup.
+# mark
+✔ Database URL … postgresql://postgres:postgres@localhost:54322/postgres
 
 # mark
-✔ Target database connection string … postgresql://postgres:postgres@localhost:54322/postgres
-📡 Connected to database with "[secret]ql://postgres:postgres@localhost:54322/postgres"
-Generated snaplet client type definitions for generate: snaplet-client.d.ts (see: https://docs.snaplet.dev/core-concepts/generate#introducing-the-snaplet-data-client)
-Generated snaplet.config.ts type definitions: snaplet.d.ts (see: https://docs.snaplet.dev/reference/configuration#type-safe-configuration-based-on-your-database-structure)
-ℹ Database structure: Create generate config...
-Generated config: /name-of-new-app/snaplet.config.ts
-Youre all set! Take a look at our docs to find out what you can do from here:
-https://docs.snaplet.dev/core-concepts/generate
-
-Generate seeds:
-You can take a look at the sql statements that can be generated with:
-snaplet generate --sql
-
-Load generated data:
-To load the generated data into your database execute the command:
-snaplet generate
+✔ Would you like to link your account from Snaplet Cloud to have more realistic data using our AI model? 🤖 … no
+
+⠹ Collecting information from your database
+For the best AI-enhanced data generation experience, create a free Snaplet account and login with: snaplet auth login.
+
+Snaplet uses details about your database to improve our generation results.
+
+⠧ Generating assets for @snaplet/seed
 ```
 
-> The source database credentials will be used to introspect your database and generate a base config
+> For this guide we assumed you have not linked your Snaplet account. To connect your account `snaplet auth setup` and follow the steps. 
 
-**4.2.** In the `snaplet.config.ts` update the `generate` statement to create 10 users and 20 todos, the todos should be assigned to at least one of the 10 users.
+**4.2.** Update the `seed.mts` file to create 10 users and 20 todos, the todos should be assigned to at least one of the 10 users.
 
-```ts snaplet.config.ts
-generate: {
-  async run(snaplet) {
-    await snaplet.$pipe([
-      // 10 users
-      snaplet.users((x) => x(10)),
-      // 20 todos, with one of the 10 users assigned to each todo
-      snaplet.todos((x) => x(20), { autoConnect: true }),
-    ]);
-  },
-},
+```ts seed.mts
+await snaplet.$pipe([
+  // create 10 users
+  snaplet.users((x) => x(10)),
+  // create 20 todos, for each of the 10 users
+  snaplet.todos((x) => x(20)),
+]);
 ```
 
-Once you have gone through all the steps, you are now ready to seed your database.
+You are now ready to seed your database.
 
 #### 5. Reset and seed your local development database
 
-**5.1.** Run the `snaplet generate` to populate the project seed script in the `supabase/seed.sql` file.
+**5.1** Run `npx tsx seed.mts` and pipe the output to your `supabase/seed.sql` file.
 
 ```bash
 snaplet generate --sql > supabase/seed.sql
 ```
```

Comment:
Text and code doesn't match. Now we use a `DRY` env variable by default to switch between "--sql", or not mode. By default it'll produce SQL.

To apply the seeds you now need to run the script with `DRY=0`